2.evaluate_data/class_balanced_log_reg_areashape_predict_sc_probabilities/process_plates

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+git_root=$(git rev-parse --show-toplevel)
```

Comment:
consider adding some more comments to this script

Reply:
Added